src/popup/router/pages/Names/AuctionList.vue

```diff
@@ -75,15 +90,61 @@ export default {
           default:
             return 1;
         }
-      }).sort(() => (this.displayMode.rotated ? -1 : 1));
+      });
     },
   },
   async mounted() {
-    this.loading = true;
     await this.$watchUntilTruly(() => this.$store.state.middleware);
-    this.activeAuctions = await this.$store.dispatch('names/fetchAuctions');
+    await this.fetchAuctionsChunk();
+    this.setScrollWatcher();
     this.loading = false;
   },
+  methods: {
+    ...mapActions('names', ['fetchAuctions']),
+    setScrollWatcher() {
+      const app = document.querySelector('#app');
+      const throttledLoadMore = () => throttle(this.checkLoadMore, 700);
+      app.addEventListener('scroll', throttledLoadMore());
+      this.$on('hook:destroyed', () => {
+        app.removeEventListener('scroll', throttledLoadMore());
+        this.isDestroyed = true;
+      });
+    },
+    async checkLoadMore() {
+      if (this.isDestroyed || !this.nextPageUrl) return;
```

Comment:
```suggestion
      if (this.isDestroyed || !this.nextPageUrl || this.loading) return;
```
according to the above suggestion